config/routes.rb

```diff
@@ -3,9 +3,19 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  get "/users" , to: "users#index"
-  get "/users/:id", to: "users#show"
-  get "/users/:id/posts", to: "posts#index"
-  get "/users/:id/posts/:post_id", to: "posts#show"
+  # get "/users" , to: "users#index"
+  # get "/users/:id", to: "users#show"
+  # get "/users/:id/posts", to: "posts#index"
+  # get "/users/:id/posts/new", to: "posts#new"
+  # post "/users/:id/posts", to: "posts#create" #create
+  # get "/users/:id/posts/:post_id", to: "posts#show"
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧